src/App.tsx

```diff
@@ -76,18 +61,14 @@ export const App: React.FC = () => {
                   </div>
                 )}
 
-                {author && loaded && !hasError && posts.length === 0 && (
+                {author && !loading && !error && posts.length === 0 && (
```

Comment:
consider moving long ternaries to a separate const with consistent name and use the const here